system/core/events.php

```diff
@@ -451,7 +451,7 @@ function notifyTimer() {
 					var next = msg.nextSibling;
 					if (msg.nodeType === 1) {
 						if(!msg.hasAttribute('time')) msg.setAttribute('time', now);
-						else if (now - msg.getAttribute('time') > 5000) msg.parentNode.removeChild(msg);
+						else if (now - msg.getAttribute('time') > 10000) msg.parentNode.removeChild(msg);
```

Comment:
This seems unrelated to styling?

Reply:
Changed commit message to "improved UX".